app/lib/actions/katello/repository/remove_content.rb

```diff
@@ -46,8 +49,9 @@ def humanized_name
         end
 
         def run
-          output[:task_id] = ForemanTasks.async_task(Actions::Katello::Repository::MetadataGenerate,
-                                                     ::Katello::Repository.find(input['repository_id'])).id
+          output[:task_id] = ForemanTasks
+            .sync_task(Actions::Katello::Repository::MetadataGenerate,
+                       ::Katello::Repository.find(input[:repository_id])).id
```

Comment:
@jlsherrill Here it is without the space change:

```rb
        def run
          output[:task_id] = ForemanTasks .sync_task(Actions::Katello::Repository::MetadataGenerate,
                                                     ::Katello::Repository.find(input[:repository_id])).id
        end
```

That comma after MetadataGenerate is sitting on column 99, the next argument must align with the open parenthesis, as dictated by Rubocop and Hound, so starting from the close square bracket (`])).id`), the line is too long.

I'm open to suggestions on how to format it differently.